src/DataTable/ActionDisplay.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useContext } from 'react';
+import BulkActions from './BulkActions';
+import TableActions from './TableActions';
+import DataTableContext from './DataTableContext';
+
+// handles layout for filters, status, and bulk actions
```

Comment:
Is this comment leftover from something? Seems like it doesn't apply entirely

Reply:
IT SURE DOES NOT. Will remove.